ihmc-avatar-interfaces/src/test/java/us/ihmc/avatar/slamTools/LevenbergMarquardtICPTest.java

```diff
@@ -105,8 +112,9 @@ public void testForwardBackwardsInputConverter()
          RigidBodyTransform reconstructedTransform = spatialInputFunction.apply(input);
 
          EuclidCoreTestTools.assertVector3DGeometricallyEquals(transformOriginal.getTranslation(), reconstructedTransform.getTranslation(), 1e-7);
-         assertTrue(AngleTools.computeAngleDifferenceMinusPiToPi(transformOriginal.getRotation().getYaw(), reconstructedTransform.getRotation().getYaw()) < 1e-7);
-//         EuclidCoreTestTools.assertEquals(transformOriginal, reconstructedTransform, 1e-7);
+         assertTrue(
+               AngleTools.computeAngleDifferenceMinusPiToPi(transformOriginal.getRotation().getYaw(), reconstructedTransform.getRotation().getYaw()) < 1e-7);
```

Comment:
Looks like additional line in here

Reply:
That's on purpose, the formatter did it so there wasn't a line overrun.